internal/pkg/template/service.go

```diff
@@ -29,12 +29,22 @@ type ServiceNestedStackOpts struct {
 	PolicyOutputs   []string
 }
 
+// SidecarOpts holds configuration that's needed if the service has sidecar containers.
+type SidecarOpts struct {
+	Name      *string
+	Image     *string
+	Port      *string
+	Protocol  *string
+	CredParam *string
```

Comment:
`RepoCreds` ?

Reply:
hmm im still more inclined to call it `CredParam`. The reason is because if you look at [this](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-repositorycredentials.html#cfn-ecs-taskdefinition-repositorycredentials-credentialsparameter), `repositoryCredentials` is actually parent of `credentialsParameter`. Since `credsParam` is the only field of `repoCreds` we flatten the nested struct. What do you think?